cypress/support/fragments/data-export/exportMappingProfile/modalSelectTransformations.js

```diff
@@ -4,7 +4,7 @@ const ModalTransformation = Modal('Select transformations');
 
 export default {
   searchItemTransformationsByName: (name) => {
-    cy.get('div[class^=modal-] input[name=searchValue]').type(`${name}{enter}`);
+    cy.get('div[class^=modal-] input[name=searchValue]').clear().type(`${name}{enter}`);
```

Comment:
A good practice is to create/extend the existed interactors to use it. Not to use cy.get directly. It is your decision to do it or not. It is just for information

Reply:
I got it